Server/fixtures/mobile.js

```diff
@@ -76,17 +100,28 @@ exports.createFixtures = async () => {
     const emptyContainer = await db.Containers.create({
       data: {
         id: 2,
-        city: "Saint-Brévin",
-        address: "Boulevard de l'Océan",
-        latitude: 47.232375,
-        longitude: -2.179429,
+        city: "Nantes",
+        address: "Rue george",
+        latitude: 47.218371,
+        longitude: -1.553621,
```

Comment:
should not be the same latitude/longitude of the first fixture

Reply:
I put it back